staging/src/k8s.io/apiserver/pkg/cel/environment/base.go

```diff
@@ -168,7 +271,34 @@ func MustBaseEnvSet(ver *version.Version) *EnvSet {
 	return entry.(*EnvSet)
 }
 
+// MustBaseEnvSetWithStrictCostConfig returns the common CEL base environments for Kubernetes for Version, or panics
+// if the version is nil, or does not have major and minor components.
+//
+// The environment involves strictCostEnforcement configuration which is applied to affected features like ValidatingAdmissionPolicy and Webhook matchConditions.
+// It is recommended to use MustBaseEnvSet instead of this function unless you are sure that you need strictCostEnforcement configurable.
+func MustBaseEnvSetWithStrictCostConfig(ver *version.Version) *EnvSet {
```

Comment:
Is this godoc and name of this function still accurate? Since both VAP and webhook match conditions now have separate feature gates, calling this function always return a env with strict cost disabled?  Should we consider a function signature like `MustBaseEnvSetWithOptions(var *version.Version, opt EnvSetOptions)` where `EnvSetOptions` has a `StrictCost` field?

Reply:
I have updated the comment with func name. Are you suggesting to wire strictCost into `MustBaseEnvSetWithOptions` and get env with StrictCost if the option is true? I don't think it would work as great as current way due to the env cache.